From 3e6b6964317aa6043431e6841d43e68740885f13 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Sat, 7 Mar 2015 14:23:38 -0500
Subject: [PATCH] login: fix copy-pasto in error path

CID #1256583.

(cherry picked from commit dcee01125dde502bd8108c36ddf2026c1348865f)
(cherry picked from commit c29a899e634e5b16c7afcce1ac7c28514988a9f6)
---
 src/login/inhibit.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/login/inhibit.c b/src/login/inhibit.c
index e2222ea69c..1deb15f33b 100644
--- a/src/login/inhibit.c
+++ b/src/login/inhibit.c
@@ -260,7 +260,7 @@ int main(int argc, char *argv[]) {
 
                 fd = inhibit(bus, &error);
                 if (fd < 0) {
-                        log_error("Failed to inhibit: %s", bus_error_message(&error, -r));
+                        log_error("Failed to inhibit: %s", bus_error_message(&error, fd));
                         return EXIT_FAILURE;
                 }
 
